source/services/dataContracts/baseResourceBuilder/baseResourceBuilder.service.ts

```diff
@@ -123,6 +123,19 @@ export interface IBaseResourceBuilder {
 		(options: IParentResourceParams<TDataType, TResourceDictionaryType>): IBaseParentDataService<TDataType, void, TResourceDictionaryType>;
 
 	/**
+	* Create a view of a parent resource with sub-resources that can be used as a base resource or
+	* as a singleton if a parent is selected
+	*/
+	createParentResourceView<TDataType extends IBaseDomainObject, TSearchParams, TResourceDictionaryType>
```

Comment:
Does this have to get called externally to build contracts / resources? Why would you call this method?